hugo/assets/scss/components/teaser.scss

```diff
@@ -65,6 +65,42 @@
         }
     }
 
+    &--blog {
+        #{ $self }__content {
+            display: flex;
+            flex-direction: column;
+            gap: 14px;
+        }   
+
+        #{ $self }__meta, #{ $self }__excerpt, #{ $self }__title {
+            margin: 0;
+        } 
+
+        #{ $self }__title {
+            @include style-heading-3;
+        }
+
+        .media {
```

Comment:
is i tcorrect we have the image class as teaser__image but the media class just as media?